File: pickcolor/src/main/java/com/betterzw/pickcolor/MainActivity.java
package com.betterzw.pickcolor;

import android.os.Bundle;

import com.stripe.android.Stripe;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);


        new Stripe(this, "pk_test_D4OIOA7CpFJzKVtHFABmYdBm00Ss9ewM29");


    }
}

File: navigation/src/main/java/com/betterzw/navigation/MainActivity.kt
package com.betterzw.navigation

import android.net.Uri
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.navigation.Navigation.findNavController

class MainActivity : AppCompatActivity(), BlankFragment.OnFragmentInteractionListener{
    override fun onFragmentInteraction(uri: Uri) {
        TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

   /* override fun onSupportNavigateUp(): Boolean {
        return findNavController(this, R.id.nav_host_fragment).navigateUp()
    }*/

    /*override fun onSupportNavigateUp()
            = findNavController(R.id.nav_host_fragment).navigateUp()*/
}
